src/displayapp/fonts/CMakeLists.txt

```diff
@@ -32,6 +32,12 @@ foreach(FONT ${FONTS})
    add_custom_target(infinitime_fonts_${FONT}
       DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${FONT}.c
    )
+
+# skip files, which currently cause problems when truning on CMake untiy build 
```

Comment:
truning?

Reply:
thx for spotting the typo.

It is fixed in the forced push